src/main/java/uk/gov/companieshouse/exception/BadRequestException.java

```diff
@@ -0,0 +1,3 @@
+package uk.gov.companieshouse.exception;
+
+public class BadRequestException extends RuntimeException {}
```

Comment:
The name of the exception should not say what response it will be. It should say what exactly happened in application, e.g. NoDirectorInRequestException. The decision what type of response it causes is made in the ExceptionHandler - this way we are not binding the exception to the response and we can change the response if needed in the future without changing the name of exception.